PerDiem/PerDiem/models/PerDiem.m

```diff
@@ -107,4 +107,13 @@ + (void)perDiemsForDate:(NSDate *)date
                         }];
 }
 
++ (BOOL) souldUsePendingTransaction: (Transaction*) transaction  day:(NSDate*) day {
+    NSDate *startOfDay = [[NSCalendar currentCalendar] startOfDayForDate:day];
+    NSDate *nextDay = [startOfDay dateByAddingDays:1];
+
+    return [transaction.future boolValue] && ([transaction.createdAt earlierDate:day] ||
```

Comment:
I'm not sure we should be using the createdAt time here. A user could potentially create a future transaction that will happen in the next month (so not in the the current period). 

I think the logic we discussed would be: 
```
if ( transactionDate is in this period && is past || (is a future && is in this period)) 
```

So all past transactions regardless of if they're futures or not, and all future "futures" in this period get added. Then we'll just change the display for past transactions that still marked as futures.